src/columns/DateTime64.jl

```diff
@@ -0,0 +1,33 @@
+is_ch_type(::Val{:DateTime64})  = true
+result_type(::Val{:DateTime64}, precission_str)  = Vector{DateTime}
+
+
+function read_col_data(sock::ClickHouseSock, num_rows::VarUInt, ::Val{:DateTime64},
+    precission_str::String)
```

Comment:
Another small typo: `precission` -> `precision` (all over this file)

Reply:
Yes, English is my pain :(